Week3/homework/step2-5.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
 function multiplyAll(arr) {
-  // eslint-disable-next-line
   let product = 1;
-
-  // Replace this comment and the next line with your code
-  console.log(arr, product);
-
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      product = product * arr[i][j];
```

Comment:
You could use a short-hand operator here:

```js
product *= arr[i][j];
```

Reply:
yes, I used it now.!